src/Nonlinear/ReverseAD/mathoptinterface_api.jl

```diff
@@ -138,27 +138,25 @@ function MOI.initialize(d::NLPEvaluator, requested_features::Vector{Symbol})
     end
     # 10 is hardcoded upper bound to avoid excess memory allocation
     max_chunk = min(max_chunk, 10)
+    max_expr_with_sub_length = max(
+        max_expr_length,
+        maximum(x -> length(x.nodes), d.subexpressions; init = 0),
```

Comment:
It's `undef` if the expression is not used in any constraint or the objective

Reply:
That's why I did [707ab59](https://github.com/jump-dev/MathOptInterface.jl/pull/2732/commits/707ab59aef5f1775fa725c735e1927a90eec2834)